core/app/models/spree/store.rb

```diff
@@ -23,6 +23,12 @@ def self.default
       where(default: true).first || new
     end
 
+    def default_cart_tax_location
+      @default_cart_tax_location ||= begin
```

Comment:
I thought because of the multiline expression. But - having written code that doesn't need it, I guess it can go here as well.